File: C/1008.c
#include <stdio.h>
int main(){
    int NF,HT;
    float VH;
    scanf("%d",&NF);
    scanf("%d",&HT);
    scanf("%f",&VH);
    printf("NUMBER = %d\n",NF);
    printf("SALARY = U$ %0.2f\n",HT*VH);
    return 0;
}

File: python/1008.py
# 1008 	Salário

cod = None
horas = None
salario = None
valorDaHora = None

def read_integer():
  try:
    # read for Python 2.x
    return int(input())
  except NameError:
    # read for Python 3.x
    return int(input())

def read_numeric():
  try:
    # read for Python 2.x
    return float(input())
  except NameError:
    # read for Python 3.x
    return float(input())


cod = read_integer()
horas = read_integer()
valorDaHora = read_numeric()
salario = "{:0.2f}".format((horas * valorDaHora))
print(str("NUMBER = ") + str(cod))
print(str("SALARY = U$ ") + str(salario))